### PR TITLE
use 'futures_cpupool' to run in parallel

### PR DESCRIPTION
**Pull request by rchaser53**
try to reduce execution time

### CODE REVIEW

**Comment by rchaser53** [comment]
im my environment

before: 0.2914115274s
after:  0.2516121114s

reduce execution time about 15%
it's a subtle

File: src/main.rs [hunk 1]
```diff
@@ -74,14 +74,7 @@ fn main() {
...
               });
-
-      let mut one = future::select_all(vec![before_thread, after_thread]);
-      while let Ok((_, _, remaining)) = one.wait() {
-          if remaining.is_empty() {
-              break;
-          }
-          one = future::select_all(remaining);
-      }
+      future::join_all(vec![before_thread, after_thread]).wait().unwrap();
```
Comment (bokuweb) [inline src/main.rs]: 👍

File: src/main.rs [hunk 2]
```diff
@@ -15,7 +18,11 @@ use image_creator::*;
 use diff::*;
 use rename::*;
 use clap::{App, Arg};
+use futures_cpupool::CpuPool;
+use futures::{Future, future};
+use std::sync::{Arc, Mutex};
 
+static RATE: f32 = 100.0 / 256.0;
```
Comment (bokuweb) [inline src/main.rs:25]: thanks!

**Review by bokuweb — APPROVED** [review]
Thanks for your PR :) It looks good 👍

**Pull request merged**